### PR TITLE
Modal component

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**Modal Component**
If user will click on button a modal or popup window will come and user can close by using buttons respectively.
Give it a try 
Documentation (4)

### CODE REVIEW

**Review by deepak29-git — COMMENTED** [review]
looks good

File: docs.html [hunk 1]
```diff
@@ -630,17 +626,19 @@ <h3 class="card-title">Our Chaning Work</h3>
...
   <div class="component-title" >Snackbar </div>
   <div class="component-detail" >
-    Snackbars/Toast inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn’t interrupt the user experience, and they don’t require user input to disappear.
+     <span  class="highlight-text" >
+      Snackbars/Toast
+    </span>   inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn’t interrupt the user experience, and they don’t require user input to disappear.
   </div>
 
   <div id="snackbar"></div>
   <div id="snackbar-center"></div>
```
Comment (amarphule) [inline docs.html:635]: Why are there empty divs? If no use you can remove it

File: docs.html [hunk 2]
```diff
@@ -738,6 +812,22 @@ <h2 class="drawer-title center">Title</h2>
...
+
+
+
+
+
+
+
+
+
+
```
Comment (amarphule) [inline docs.html:826]: remove empty space, use proper indentation. you can use a **Prettier extension** for that.

File: scripts/componentScript.js [hunk 3]
```diff
@@ -49,3 +49,27 @@ function showMessageTop(msg) {
...
+// --Responsive Navigation Script --//
+const toggleButton = document.querySelector(".toggle-button");
+const navbarLinks = document.querySelector(".navbar-links");
+
+toggleButton.addEventListener("click", () => {
+  navbarLinks.classList.toggle("active");
+});
+
+console.log(toggleButton);
+console.log(navbarLinks);
```
Comment (amarphule) [inline scripts/componentScript.js:62]: please remove console log.

File: style/docs.css [hunk 4]
```diff
@@ -185,3 +189,8 @@ hr {
 .warning {
   background-color: #ff9800;
 }
```
Comment (amarphule) [inline style/docs.css:191]: In some places, you used variables for color, here you used hex value please use one consistent methode

**Review by amarphule — CHANGES_REQUESTED** [review]
The code looks good. given some suggestions please check

File: docs.html (hunk 1 above)
Comment (tanishk15sharma) [reply docs.html:635]: This divs are for snackbar they will be visible only one click and for 5 seconds and linked to javascript so nothing to write in `div` element

File: docs.html (hunk 2 above)
Comment (tanishk15sharma) [reply docs.html:826]: Yes I am learning how to use prettier

File: scripts/componentScript.js (hunk 3 above)
Comment (tanishk15sharma) [reply scripts/componentScript.js:62]: Okay I will do the changes

File: style/docs.css (hunk 4 above)
Comment (tanishk15sharma) [reply style/docs.css:191]: okay

File: docs.html (hunk 1 above)
Comment (amarphule) [reply docs.html:635]: Okay.

**Pull request merged**